tsconfig.json

```diff
@@ -11,7 +11,10 @@
     "typeRoots": ["./node_modules/@types"],
     "allowJs": true,
     "allowSyntheticDefaultImports": true,
-    "allowUmdGlobalAccess": true
+    "allowUmdGlobalAccess": true,
+    "paths": {
+      "vexflow": ["./node_modules/vexflow/entry/vexflow-core-with-gonville.ts"]
```

Comment:
OSMD only uses tsconfig.json for generating their d.ts files, I believe. You can check with Simon for the details.